app/models/facility_group.rb

```diff
@@ -39,6 +48,8 @@ def toggle_diabetes_management
       set_diabetes_management(true)
     elsif diabetes_enabled?
       set_diabetes_management(false)
+    else
+      true
```

Comment:
This is an unrelated fix to `toggle_diabetes_management`. It returns a `nil` if there is no change, which is broken behaviour.